migrations/1_initialize_schema.up.sql

```diff
@@ -231,6 +231,7 @@ CREATE TABLE integration
   type          INTEGER REFERENCES integration_type (id) ON DELETE CASCADE,
   enabled       BOOLEAN                 NOT NULL,
   params        JSONB                   NULL,
+  creator       VARCHAR                 NOT NULL,
```

Comment:
we have 'owner' for other entities 
maybe it's a good idea to follow that rule, isn't it?